src/storage/mvcc/reader/reader.rs

```diff
@@ -531,24 +530,27 @@ impl<S: EngineSnapshot> MvccReader<S> {
         Ok((locks, false))
     }
 
-    /// Scan writes that satisfies `filter(key)` returns true in the key range
-    /// [start, end). At most `limit` locks will be returned. If `limit` is
-    /// set to `0`, it means unlimited.
+    /// Scan the writes to get all the latest keys with their corresponding
+    /// write records at the given version, if the key does not exist at
+    /// that point, an `Option::None` will be placed. The return type is
+    /// `(Vec<(key, commit_ts, Option<old_write>)>, has_remain)`.
+    ///   - `key` is the encoded key without commit ts.
+    ///   - `commit_ts` is the latest commit ts of the key.
+    ///   - `old_write` is the write record at the given version.
+    ///   - `has_remain` indicates whether there MAY be remaining writes that
+    ///     can be scanned.
     ///
-    /// The return type is `(writes, has_remain)`. `has_remain` indicates
-    /// whether there MAY be remaining writes that can be scanned.
-    pub fn scan_writes<F>(
+    /// This function is used by `txn::commands::FlashbackToVersionReadPhase`
+    /// and `txn::commands::FlashbackToVersion` to achieve the MVCC
+    /// overwriting.
+    pub fn scan_old_writes(
```

Comment:
cc @you06 @Connor1996 I implemented this function to get the old MVCC records for all keys, it will handle the `WriteType::Lock` and `WriteType::Rollback` as well in this function. PTAL, thx a lot!